www/autonCreator.js

```diff
@@ -489,66 +497,68 @@ function autonCreatorDrawLoop() {
                 let otherPath = paths[i];
                 let otherWaypointIndex = otherPath.getWaypointIndexByName(selectedWaypoint.name);
 
-                if (otherWaypointIndex !== undefined) {
-                    let otherWaypoint = otherPath.getWaypoint(otherWaypointIndex);
-
-                    otherWaypoint.x = selectedWaypoint.x;
-                    otherWaypoint.y = selectedWaypoint.y;
-                    otherWaypoint.angle = selectedWaypoint.angle;
-                    sharedWaypoints[sharedIndex].x = selectedWaypoint.x;
-                    sharedWaypoints[sharedIndex].y = selectedWaypoint.y;
-                    sharedWaypoints[sharedIndex].angle = selectedWaypoint.angle;
-                    if (isTank) {
-                        otherWaypoint.spline_angle = selectedWaypoint.spline_angle;
-                        sharedWaypoints[sharedIndex].spline_angle = selectedWaypoint.spline_angle;
-                    }
-
-                    if (otherPath.getNumWaypoints() > 0) {
-                        // Draw waypoints
-                        let waypoints = otherPath.getWaypoints();
-
-                        for (let waypoint of waypoints) {
-                            let waypointPos = inchesToPixels(new point(waypoint.x, waypoint.y));
-                            let waypointRotation = waypoint.angle;
-                            fieldContext.save();
-                            fieldContext.translate(waypointPos.x, waypointPos.y);
-                            fieldContext.rotate(toRadians(waypointRotation + 90));
-                            fieldContext.globalAlpha = 0.5;
-                            fieldContext.drawImage(robotImage, Math.floor(-robotWidthPxl * .5), Math.floor(-robotCenterPxl), Math.floor(robotWidthPxl), Math.floor(robotHeightPxl));
-                            fieldContext.restore();
-                        }
-                    }
+                if (otherWaypointIndex === undefined) {

```

Comment:
Are these the same changes from pull request #38 ? If so, we should merge that one in and rebase this branch (so that we aren't trying to add two branches with the same changes in slightly different locations - this may cause some issues)

Reply:
#38 Should be rebased now.